features/steps/steps.py

```diff
@@ -221,3 +249,41 @@ def step_impl(context, related, relating, other_entity, condition):
                         errors.append(instance_structure_error(inst, rel.RelatingObject))
 
     handle_errors(context, errors)
+
+@then('The relative placement of that {entity} must be provided by an {other_entity} entity')
+def step_impl(context, entity, other_entity):
+    if getattr(context, 'applicable', True):
+        errors = []
+        for obj in context.instances:
+            if not obj.ObjectPlacement.is_a(other_entity):
+                errors.append(instance_placement_error(obj, other_entity, "", "", "", ""))
+        handle_errors(context, errors)
```

Comment:
Formal parameter `{entity}` isn't used anywhere - consider removing it.

Reply:
The intention was to have a little bit more flexibility/reusability in the .feature file, allowing the use of whatever word is required, without forcing the use of specifically `IfcElement`. 

I don't think it's a huge benefit, as the sentence is quite rigid, but perhaps it's worth to keep it?